app/controllers/projects_controller.rb

```diff
@@ -17,6 +22,27 @@ def create
     end
   end
 
+  def edit
+    @employees = Employee.all
+    @project = Project.find(params[:id])
+    respond_to do |format|
+      format.js {}
+    end
+  end
+
+  def update
+    @project = Project.find(params[:id])
+    if @project.valid?
+      @project.update(project_params)
+      if params["Projectemployee"].present?
```

Comment:
Here also I am updating project details and saving newly selected employees for that project.

For edit action I am sending Js request on click of project card. To display respective project details in edit for I need @project = Project.find(params[:id]). And I have to display employees name list in 'Employees' field of the same form so I need @employees = Employee.all.